.travis.yml

```diff
@@ -72,11 +72,13 @@ matrix:
         # time.
 
         - os: linux
-          env: PYTHON_VERSION=3.4 NUMPY_VERSION=1.9
+          env: PYTHON_VERSION=3.5
```

Comment:
this one can be dropped as well I think, it should be enough to test python3.5 once in the matrix